app/assets/stylesheets/style.scss

```diff
@@ -1,7 +1,8 @@
 body {
   margin:0px;
+  background-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.